package.json

```diff
@@ -24,7 +24,8 @@
     "test-jest-watch": "jest --env=jsdom --watch -o",
     "test-ui": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe chrome ./**/ui.test.js",
     "test-ui-hl": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe 'chrome:headless' ./**/ui.test.js",
-    "new": "node scripts/create-a-m-o/index.js"
+    "new": "node scripts/create-a-m-o/index.js",
+    "deploy-branch": "npm run build-storybook && SOURCE_BRANCH_NAME=\"$(git rev-parse --abbrev-ref HEAD)\" bash scripts/branch-deployment.sh --my-credentials && SOURCE_BRANCH_NAME=\"$(git rev-parse --abbrev-ref HEAD)\" bash scripts/branch-deployment-notifier.sh"
```

Comment:
Is there a command to get the actual branch name to replace `SOURCE_BRANCH_NAME`?

Reply:
Yes, `git rev-parse --abbrev-ref HEAD` ;)

This command works standalone.

Reply:
Ah,... SOURCE_BRANCH_NAME is a global variable. Perfect.

Reply:
No, the SOURCE_BRANCH_NAME variable is being assigned, directly before the npm script is being executed. And the value will be whatever `git rev-parse --abbrev-ref HEAD` yields. And this command yields the current branch you are working on.

Reply:
Global variables should be the SLACK_TOKEN variable. Because the deployment script will need it to post to the #plib-deployments channel.